olm_deploy/Dockerfile.registry

```diff
@@ -0,0 +1,17 @@
+FROM quay.io/operator-framework/upstream-registry-builder:latest as builder
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13
+
+WORKDIR /
+ARG MANIFEST_BUNDLE
+COPY ${MANIFEST_BUNDLE} /manifests
+
+# Need to remove the art.yaml and image-references files from the list of OLM
+# manifests as those are invalid in a manifest bundle
+RUN chmod -R g+w /manifests && rm /manifests/art.yaml /manifests/*/image-references
+COPY olm_deploy/scripts/ /scripts/
+
+COPY --from=builder /bin/initializer /usr/bin/initializer
+COPY --from=builder /bin/registry-server /usr/bin/registry-server
+COPY --from=builder /bin/grpc_health_probe /usr/bin/grpc_health_probe
+
+WORKDIR /bundle
```

Comment:
Something we may want to consider further down the line is providing a better developer workflow, as in take a list of arguments for the various operator and operand images and do the CSV manipulation inside of the Dockerfile and instantiate and serve that registry bundle from Dockerfile.

In this case, we may need to extend the e2e suite or the deploy-metering utility to create a CatalogSource that points to an external image registry, instead of the current implementation which references a local registry service.

Reply:
Is the naming of this file something required by OLM, or I don't understand the naming?

Reply:
The naming is arbitrary - we would just reference this Dockerfile in the release repository as part of our configuration, and other places in the repository such as hack scripts and Makefile targets.